packages/salesforcedx-vscode-apex/src/commands/forceLaunchApexReplayDebuggerWithCurrentFile.ts

```diff
@@ -10,53 +10,65 @@ import {
   CommandBuilder,
   EmptyParametersGatherer,
   fileExtensionsMatch,
-  fileUtils,
   notificationService,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from '@salesforce/salesforcedx-utils-vscode';
 import * as vscode from 'vscode';
+import { channelService } from '../channels';
 import { nls } from '../messages';
 import { testOutlineProvider } from '../views/testOutlineProvider';
 import { forceAnonApexDebug } from './forceAnonApexExecute';
 
 export async function forceLaunchApexReplayDebuggerWithCurrentFile() {
-  const editor = vscode.window.activeTextEditor;
-  if (!editor) {
+  try {
+    const editor = vscode.window.activeTextEditor;
```

Comment:
Lines 26-60 in the new version are lines 25-59 in the old version.  No code changes were made here, other than wrapping in a `try` block and indenting.